conf.yaml

```diff
@@ -1150,25 +1150,53 @@ contents:
                         repo:   apm-agent-rum-js
                         path:   CHANGELOG.asciidoc
                         exclude_branches:   [ 3.x, 2.x, 1.x, 0.x ]
-          - title:      ECS Logging Java Reference
-            prefix:     en/ecs-logging/java
-            current:    1.x
-            branches:   [ master, 1.x, 0.x ]
-            live:       [ master, 1.x ]
-            index:      docs/index.asciidoc
-            chunk:      1
-            tags:       ECS-logging/java/Guide
-            subject:    ECS Logging Java Reference
-            sources:
-              -
-                repo:   ecs-logging-java
-                path:   docs
-              -
-                repo:   docs
-                path:   shared/versions/stack/{version}.asciidoc
-              -
-                repo:   docs
-                path:   shared/attributes.asciidoc
+          - title:      ECS logging
+            base_dir:   en/ecs-logging
+            sections:
+              - title:      ECS Logging Overview
+                prefix:     overview
+                current:    master
+                branches:   [ master ]
+                live:       [ master ]
```

Comment:
We don't need to include `live` if it's the same as the full list of `branches` but it also doesn't hurt.

Reply:
I kept it in because it eventually won't be the only branch